src/Infrastructure/AmountParser.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Infrastructure;
+
+/**
+ * @licence GNU GPL v2+
+ * @author Kai Nissen < kai.nissen@wikimedia.de >
+ * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ */
+class AmountParser {
+
+	private $locale;
+
+	public function __construct( string $locale = 'de_DE' ) {
```

Comment:
Hmm, sure. Beware that it is you that added the optional thing here, I just moved it around

```php
public function setAmountFromString( string $value, string $locale = 'de_DE' ) {
```